trinity/__init__.py

```diff
@@ -1,7 +1,11 @@
 import pkg_resources
 
 # TODO: update this to use the `trinity` version once extracted from py-evm
-__version__: str = pkg_resources.get_distribution("py-evm").version
+try:
+    __version__: str = pkg_resources.get_distribution("trinity").version
+except pkg_resources.DistributionNotFound:
+    # mypy doesn't like that `__version__` is defined twice
+    __version__: str = pkg_resources.get_distribution("py-evm").version  # type: ignore
```

Comment:
How about defining `__version__` *before* the `try` instead of silencing mypy?

Reply:
Do mean a bare type def'n, like:

```py
__version__: str
try:
    __version__ = pkg_resources.get_distribution("trinity").version
except pkg_resources.DistributionNotFound:
    # mypy doesn't like that `__version__` is defined twice
    __version__ = pkg_resources.get_distribution("py-evm").version
```

Reply:
All of these are equally ugly in my opinion so unless someone feel strongly leaving as is.  Stop-gap solution until this lives by itself in it's own repo.

Reply:
No strong feeling at all.